src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIMeleeGuard.java

```diff
@@ -24,6 +29,11 @@
 public class EntityAIMeleeGuard extends AbstractEntityAIGuard
 {
     /**
+     * The max distance an entity may be for the guard to help
+     */
+    private static final double MAX_DISTANCE_TO_HELP_GUARD = 60;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "MAX_DISTANCE_TO_HELP_GUARD" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)